acousticbrainz/views/index.py

```diff
@@ -7,6 +7,8 @@
 @index_bp.route("/")
 def index():
     stats, last_collected = get_stats()
+    for key in stats:
+        stats[key] = "{:,}".format(stats[key])
```

Comment:
It's https://docs.python.org/2/library/string.html#format-specification-mini-language. In CritiqueBrainz we use something similar to Django stuff: https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/frontend/views.py#L27-L28, but we have support for localization there and additional libraries that provide this functionality.